fdbserver/LogSystem.cpp

```diff
@@ -272,6 +272,15 @@ void LogSet::getPushLocations(VectorRef<Tag> tags, std::vector<int>& locations,
 	//	.detail("Included", alsoServers.size()).detail("Duration", timer() - t);
 }
 
+LogPushData::LogPushData(Reference<ILogSystem> logSystem, int tlogCount) : logSystem(logSystem), subsequence(1) {
+	ASSERT(tlogCount > 0);
+	messagesWriter.reserve(tlogCount);
```

Comment:
Pre-allocating this vector and going over the local tlogs only - these are the improvements in this PR, right?

Reply:
This is not the main improvement. This PR mainly uses the pre-calculated `tlogCount` here, thus saving the time to construct `LogSystemConfig`.

Reply:
> uses the pre-calculated tlogCount here
Probably that's what I meant by "going over the local tlogs". Anyway, I got it.